admin/helpers/hdf-presenter.js

```diff
@@ -4,7 +4,7 @@ const moment = require('moment')
 const hdfPresenter = {}
 
 hdfPresenter.getResultsDate = (hdf) => {
-  return moment(hdf.checkEndDate).add(1, 'weeks').isoWeekday('Monday') // next monday after the check ends
+  return moment(hdf.checkEndDate).add(1, 'weeks').isoWeekday('Monday').set({ hour: 8, minutes: 0, seconds: 0 }) // next monday after the check ends
```

Comment:
@danwilliams does the checkEndDate need a moment wrapper?
I was under the impression it is already 'momentised' at that level.